sdk/js-query/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "declaration": true,
+    "outDir": "./lib/esm",
+    "strict": true,
+    "esModuleInterop": true,
+    "downlevelIteration": true,
+    "allowJs": true,
+    "lib": ["dom", "es5", "scripthost", "es2020.bigint"]
+  },
+  "include": ["src"],
+  "exclude": ["node_modules", "**/*.test.*", "**/__tests__/*"]
```

Comment:
Question: Do we normally exclude "lib/*"?  I'm fuzzy on tsconfig stuff.

Reply:
We're only going to include source files in the `src` directory here so we don't need to exclude `lib`